### ISSUE TITLE
click-outside directive on iOS only working if clicking on inputs

### ISSUE THREAD

**Issue by gwildu** [BosNaufal/vue-click-outside#1]
When you try the demo on an iOS device, it is only working if you click on an input field.

I guess it is related to what is explained here: http://www.shdon.com/blog/2013/06/07/why-your-click-events-don-t-work-on-mobile-safari.

**Comment by BosNaufal** [comment]
Hi @gwildu, thank you for giving me an issue. But I'm sorry can't help you. Because I have no IOS device to work on right now. :confused: 

Your contribution will be very helpful~

**Comment by gwildu** [comment]
Currently I don't have time to work on that, maybe in some week it looks different...

**Comment by mediafreakch** [comment]
@gwildu This is not the package available as `vue-click-outside` on npmjs.com!

**Comment by andreiglingeanu** [comment]
@mediafreakch Yeah, I've also noticed that and I'd like to use that package and receive updates to it!

@BosNaufal Would you mind taking some time to publish it to npmjs.com? I'm able to give you a hand of help, if you need any, just ping me!

_UPDATE: I've attached a pull request that adds a package.json to that project._

**Comment by andreiglingeanu** [comment]
Actually, I discovered that vue-click-outside is not compatible with Vue 2.0. You'll be better off using [vue-clickaway](https://github.com/simplesmiler/vue-clickaway).
